### PR TITLE
Photostack flipping images fix

### PR DESCRIPTION
**Pull request by KamilWo**
Changed events and timeouts.

### CODE REVIEW

**Comment by KamilWo** [comment]
Please, could you not merge those changes yet, Martyna will test them in action.

File: addons/bestja_frontend_ucw/static/src/js/photostack.js [hunk 1]
```diff
@@ -230,49 +230,104 @@
...
-	Photostack.prototype._flip = function() {
+		Photostack.prototype._flip = function() {
 		if (window.innerWidth < 767 ){
-			return false;
+                    return false;
 		}
 		var self = this;
 		var a = [].slice.call(document.getElementsByTagName('figure'));
-		var timeStart, timeEnd, timeEndOut;
+                //timeMouseMoveStart, timeEnd are helping to prevent events during flips
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_ucw/static/src/js/photostack.js]: Wrong indentation. Also there should be a space after `//`.

Comment (KamilWo) [inline addons/bestja_frontend_ucw/static/src/js/photostack.js]: Intentation changed and space added. Thank you.

**Comment by KrzysiekJ** [comment]
Merged in 00bb809.

**Pull request closed without merging**